data/xapleaux/xapleaux missions.txt

```diff
@@ -0,0 +1,192 @@
+			`	This time the tablet depicts a ship being burned on the exterior, landing, and then a number of aliens coming forward and spraying something onto damaged parts before allowing it to take off again. The same question is posed to you and, realizing that you don't actually know how to say 'no' at this point, you make the same two sharp sounds again.`
+			`	The alien gives another click of apparent satisfaction and turns a little to those waiting nearby. All six of its arms come to life and you are granted the unsettling realization that its beak, which you thought had only two parts, actually had four - two of which now unfolded into articulated mandibles which click and snap rapidly back and forth in, out, and around a mouth full of grotesque moving parts too difficult to name in the moment. Its entire being seems to suddenly be in motion as it emits a rapid series of clicks, trills, shrieks and whistles, gestures in a complex sequence with all of its arms, and seems to alternately inflate, deflate, or wiggle as the fronds of 'foliage' dance and sway in a pattern clearly independent of what its arms and head are doing.`
+			choice
+				`	(Panic and punch it.)`
+				`	(Take a step back and a deep breath.)`
+					goto calm
+			`	As your fist wildly connects with the alien's carapace it recoils from you, though you get the sense it was more from shock than from any force you yourself actually managed to impart. Everything comes to a sudden halt as every last one of them looks straight at you - all of them frozen in place, in some cases mid-step, to see what happens next.`
+			`	You realize that, regardless of what you attempt, you are unlikely to achieve anything except by apology.`
+			choice
+				`	(Raise your hands peaceably.)`
+			`	Events resume.`
+			label calm
+			`	The assembled aliens spring into motion, carrying equipment and hoses towards your ships and proceeding to clamber more or less all over it, inspecting it for heat damage and refuelling it, hopefully with something your ship can handle.`
+
+#This is only one of various future possible paths to unlock access to them, but for now it's the only one with requirements already in-game.
+			branch automata
+				has "event: wanderers: exiles have drones"
+				has "event: remnant: sign studies complete"
+			`	When they are done the same alien approaches you again and holds out a weighty gemstone to you. It is milky and translucent, and a brilliant shade of pink; cut into an octagonal shape with layers of surface faceting it scatters diffuse pink patterns across the ground as the brilliance of the core shines down through it. The alien points at you, and then at the crystal, and mimes holding it out to you before pointing at itself. It then points at the tablet which shows two animated symbols: an empty box filling, and a blob spraying something onto a panel marked with a fire symbol.`
```

Comment:
Are you planning to have the crystal as some sort of special outfit or license? Seems natural if the player owns it now.

Reply:
It will likely feature as something. In this particular scene it is merely being used as a trivial object to visually represent the idea that the animations are associated with you receiving something.
The fact that it was something conveniently to hand though, will certainly come up.